src/util.h

```diff
@@ -114,6 +114,12 @@ bool Truncate(const string& path, size_t size, string* err);
 #define PATH_MAX _MAX_PATH
 #endif
 
+/// ReplaceContent the content of \a file_dst with the content of \a new_content.
+/// The file pointed to by \a file_dst will keep the same uid and gid.
+/// In case of an error, false is returned and \a err will carry the error message.
+/// On Windows, the same gid / uid is not garanteed
```

Comment:
I would merge this sentence with the second one like this: On platforms other than Windows, the file pointed to by \a file_dst will keep the same uid and gid.

typo in garanteed, but that wouldn't matter then ;)